pkg/rehearse/metrics.go

```diff
@@ -21,9 +21,10 @@ type Metrics struct {
 	Repo string `json:"repo"`
 	Pr   int    `json:"pr"`
 
-	ChangedCiopConfigs []string `json:"changed_ciop_configs"`
-	ChangedPresubmits  []string `json:"changed_presubmits"`
-	ChangedTemplates   []string `json:"changed_templates"`
+	ChangedCiopConfigs     []string                `json:"changed_ciop_configs"`
+	ChangedPresubmits      []string                `json:"changed_presubmits"`
+	ChangedTemplates       []string                `json:"changed_templates"`
+	ChangedClusterProfiles []config.ClusterProfile `json:"changed_cluster_profiles"`
```

Comment:
Can we store just names here to keep the output easier to eyball?

Reply:
Ah, `ClusterProfile` used to be just `type ClusterProfile string`, I forgot to change it when I added the other field.

Reply:
Forgot to mention: changed in 323f9d02f0612bf84e219de2e65edefafa4d6619.